RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -2006,4 +2021,102 @@ Bool_t AliAnalysisTaskAO2Dconverter::Select(TParticle *part, Float_t rv, Float_t
 
 } // AliAnalysisTaskAO2Dconverter::Select(TParticle* part, Float_t rv, Float_t zv)
 
+//_________________________________________________________________________________________________
+AliAnalysisTaskAO2Dconverter::FwdTrackPars AliAnalysisTaskAO2Dconverter::MUONtoFwdTrack(AliESDMuonTrack &MUONTrack) {
+  // Convert MCH Track parameters and covarianceS matrix to the RUN3 Forward track coordinate system
+
+
+  FwdTrackPars convertedTrack;
+
+  // Parameter conversion
+  double alpha1, alpha3, alpha4, x2, x3, x4;
+
+  alpha1 = TMath::Tan(MUONTrack.GetThetaX());
+  alpha3 = TMath::Tan(MUONTrack.GetThetaY());
+  alpha4 = MUONTrack.GetInverseBendingMomentum();
+
+  x2 = TMath::ATan2(-alpha3, -alpha1);
+  x3 = -1. / TMath::Sqrt(alpha3 * alpha3 + alpha1 * alpha1);
+  x4 = alpha4 * -x3 * TMath::Sqrt(1 + alpha3 * alpha3);
+
+  // Set output parameters
+  convertedTrack.fX = AliMathBase::TruncateFloatFraction(MUONTrack.GetNonBendingCoor(), mMuonTrNonBend);
+  convertedTrack.fY = AliMathBase::TruncateFloatFraction(MUONTrack.GetBendingCoor(), mMuonTrBend);
+  convertedTrack.fZ = AliMathBase::TruncateFloatFraction(MUONTrack.GetZ(), mMuonTrZmu);
+  convertedTrack.fPhi = AliMathBase::TruncateFloatFraction(x2, mMuonTrThetaX);
+  convertedTrack.fTgl = AliMathBase::TruncateFloatFraction(x3, mMuonTrThetaX);
+  convertedTrack.fSigned1Pt = AliMathBase::TruncateFloatFraction(x4, mMuonTr1P);
+  convertedTrack.fChi2 = AliMathBase::TruncateFloatFraction(MUONTrack.GetChi2(), mMuonTrCov);
+  convertedTrack.fChi2MatchMCHMID = AliMathBase::TruncateFloatFraction(MUONTrack.GetChi2MatchTrigger(), mMuonTrCov);
```

Comment:
Would it be possible to also fill the RAtAsorberEnd and PDCA branches? The first is easy:

convertedTrack.fRAtAbsorberEnd = AliMathBase::TruncateFloatFraction(MUONTrack.GetRAtAbsorberEnd(), mMuonTrCov);

For the p x dca, I tried locally:
    double pdca;
    pdca= MUONTrack.P() * MUONTrack.GetDCA();
    convertedTrack.fPDca = AliMathBase::TruncateFloatFraction(pdca, mMuonTrCov);

which works.